### PR TITLE
brew-*: add French translation

### PR DESCRIPTION
**Pull request by quentin7b**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

--- 

Translations for
- brew
- brew-cask
- brew-bundle

### CODE REVIEW

File: pages.fr/common/brew-bundle.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# brew bundle
+
+> Outil de gestion de paquets pour Homebrew, Homebrew Cask et le Mac App Store.
```
Comment (kbdharun) [inline pages.fr/common/brew-bundle.md:3]:
```suggestion
> Bundler pour Homebrew, Homebrew Cask et le Mac App Store.
```

**Review by kbdharun — APPROVED** [review]
The translations LGTM, Thanks for your contribution. I have a suggestion for this page.

> I will wait for reviews from native speakers.

File: pages.fr/common/brew-bundle.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+> Plus d'informations : <https://github.com/Homebrew/homebrew-bundle>.
+
+- Installe tous les paquets listés dans le Brewfile situé dans le chemin courant :
```
Comment (patricedenis) [inline pages.fr/common/brew-bundle.md:6]:
```suggestion
- Installe tous les paquets listés dans le Brewfile situé dans le dossier courant :
```

File: pages.fr/common/brew-bundle.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`brew bundle`
+
+- Installe tous les paquets listés dans le Brewfile situé au chemin spécifié :
+
+`brew bundle --file={{chemin/du/fichier}}`
```
Comment (patricedenis) [inline pages.fr/common/brew-bundle.md:12]: We should normalize mentions to path files as described in [here](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md). 
```suggestion
`brew bundle --file={{chemin/vers/fichier}}`
```

File: pages.fr/common/brew-cask.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+- Désinstalle un cask :
+
+`brew uninstall --cask {{nom_du_cask}}`
+
+- Désinstalle un cask et supprime les paramètres et fichiers associés :
+
+`brew zap --cask {{nom_du_cask}}`
+
+- Affiche les informations in cask en particulier :
```
Comment (patricedenis) [inline pages.fr/common/brew-cask.md:35]:
```suggestion
- Affiche les informations d'un cask en particulier :
```

File: pages.fr/common/brew.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+- Met à jour une formule ou cask déjà installé (si rien n'est précisé, toutes les formules et tous les casks installés seront mis à jour) :
+
+`brew upgrade {{formule}}`
+
+- Récupère la dernière version d'Homebrew et toutes les formules et casks depuis le dépôt source d'Homebrew :
+
+`brew update`
+
+- Montre les formules et les casks avec une nouvelle version disponible :
```
Comment (patricedenis) [inline pages.fr/common/brew.md:22]:
```suggestion
- Montre les formules et les casks disposants d'une nouvelle version :
```

File: pages.fr/common/brew.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+`brew outdated`
+
+- Recherche une formule (c.a.d. un package) et/ou un casks (c.a.d une application native) :
```
Comment (patricedenis) [inline pages.fr/common/brew.md:26]:
```suggestion
- Recherche une formule (c.a.d. un package) et/ou un cask (c.a.d une application native) :
```

**Review by patricedenis — CHANGES_REQUESTED** [review]
Thanks for your contribution. I have made several suggestion.

File: pages.fr/common/brew-bundle.md (hunk 1 above)
Comment (quentin7b) [reply pages.fr/common/brew-bundle.md:3]: I'm not quite sure "Bundler" will be easily understood by french speaking users...
Maybe a second opinion can help us on this ?

**Review by patricedenis — APPROVED** [review]
Thank you.
With two approvals, it's ready to be merged, congratulations !!

**Review by kbdharun — APPROVED** [review]

**Pull request merged**